packages/strapi-plugin-upload/admin/src/utils/formatBytes.js

```diff
@@ -1,17 +1,13 @@
-// Source: https://stackoverflow.com/questions/15900485/correct-way-to-convert-size-in-bytes-to-kb-mb-gb-in-javascript
-function formatBytes(receivedBytes, decimals) {
-  const bytes = receivedBytes * 1000;
+import byteSize from 'byte-size';
```

Comment:
Where is this lib installed?

Reply:
Just added in the plugin, I thought it was already installed!